translations/nm-tray_de.ts

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de">
+<context>
+    <name>ConnectionInfo</name>
+    <message>
+        <location filename="../src/connectioninfo.ui" line="14"/>
+        <source>Connection information</source>
+        <translation>Verbindungsinformationen</translation>
+    </message>
+</context>
+<context>
+    <name>NmList</name>
```

Comment:
The NmList dialog is just debug preview of known NetworkManager objects.... once the project is mature, it will probably be dropped.

Just FYI